models/desired_lrp_test.go

```diff
@@ -83,7 +83,8 @@ var _ = Describe("DesiredLRP", func() {
               },
               "user": "vcap",
               "log_source": "APP",
-			  "suppress_log_output": false
+			  "suppress_log_output": false,
+              "volume_mounted_files": null
```

Comment:
I see that this tabbing was weird before you were in the file, but can you fix it since you are in this file?